common/pinot/pinotQueryValidator.go

```diff
@@ -453,21 +453,22 @@ func processCustomKeyword(operator string, colNameStr string, colValStr string)
 		colNameStr, operator, colValStr, colNameStr, operator, colValStr)
 }
 
-func processCustomString(comparisonExpr *sqlparser.ComparisonExpr, colNameStr string, colValStr string) string {
-	// change to like statement for partial match
-	comparisonExpr.Operator = sqlparser.LikeStr
```

Comment:
This part was a legacy code from the first version of pinot_query_validator. Forgot to delete previously.